lib/rubocop/cop/rspec/describe_symbol.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Avoid passing a Symbol as the first argument to `describe`.
+      #
+      # @example
+      #   # bad
+      #   describe :my_method do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   describe '#my_method' do
+      #     ...
+      #   end
+      #
+      # See https://github.com/rspec/rspec-core/issues/1610
+      class DescribeSymbol < Cop
+        MSG = 'Avoid passing a Symbol as the first argument to ' \
+              '`describe`.'.freeze
+
+        def_node_matcher :describe_symbol?, <<-PATTERN
+          (send {(const nil :RSpec) nil} :describe $sym ...)
+        PATTERN
+
+        def on_send(node)
+          match = describe_symbol?(node)
```

Comment:
I think you can just do `describe_symbol?(node) { add_offense(node, :expression }` since node and match should be the same and the block will only trigger when you match.